### ISSUE TITLE
What resolution to specify

### ISSUE THREAD

**Issue by Masaaki-75** [S-aiueo32/lpips-pytorch#18]
i wonder what are the default resolutions for the pretrained alex-net, squeeze-net and vgg16-net. Will it be okay if i apply the lpips loss directly to the images whose resolutions are different from those of the training images for the above networks?
